portal-web/test/functional/com/liferay/portalweb/tests/enduser/wcm/exportimport/ExportImport.testcase

```diff
@@ -305,6 +305,107 @@
 		</execute>
 	</command>
 
+	<command name="ExportImportDMRootFolder" priority="5">
```

Comment:
@brianwulbern I noticed that this test appears in ee-7.0.x but does not appear in master. It looks like this test was removed under ec6a7bcee499b0a6fe9736825db86551ac147d5a as part of [LRQA-30250](https://issues.liferay.com/browse/LRQA-30250). How do you want to proceed with this?